docs/content/primary-key-table/merge-engine.md

```diff
@@ -247,6 +247,14 @@ Current supported aggregate functions and data types are:
   The first_non_null_value function selects the first non-null value in a data set.
   It supports all data types.
 
+* `rbm32`:
+  The rbm32 function aggregates multiple serialized 32-bit RoaringBitmap into a single RoaringBitmap.
+  It supports VARBINARY data type.
+
+* `rbm64`:
```

Comment:
do we really need rbm64? Maybe 32 is enough?

Reply:
I think it depends on the user's case. If the input value is already normalized in [0, Integer.MAX_VALUE], then rbm32 is enough. Otherwise, the rbm64 is required.